CONTRIBUTING.md

```
@@ -51,14 +51,14 @@ Releases are deliberate, not automatic. `scripts/release.sh` bumps the version,
 #### 1. Set yourself up to start coding
 
-- **1.1. Pick an issue**: Select one from the project GitHub repository [issue list](https://github.com/yeagerai/genlayer-cli/issues) and assign it to yourself.
+- **1.1. Pick an issue**: Select one from the project GitHub repository [issue list](https://github.com/genlayerlabs/genlayer-cli/issues) and assign it to yourself.
 
 - **1.2. Create a branch**: create the branch that you will work on by using the link provided in the issue details page (right panel at the bottom - section "Development")
 
 - **1.3. Setup the CLI locally**: clone the repository and install dependencies
 
    ```sh
-   $ git clone https://github.com/yeagerai/genlayer-cli.git
+   $ git clone https://github.com/genlayerlabs/genlayer-cli.git
```

markdownlint-cli2 (0.23.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)